org.alloytools.alloy.dash/src/main/java/ca/uwaterloo/watform/rapidDash/DashPythonTranslation.java

```diff
@@ -659,6 +659,38 @@ public List<State> getAllStates() {
         return states;
     }
 
+    public List<Transition> getAllTransitions() {
+        List<Transition> transitions = new ArrayList<>();
+
+        for(State state: statesMap.values()) {
+            for(Transition t: state.transitions) {
+                transitions.add(t);
+            }
+        }
+        return transitions;
+    }
+
+    public List<OrthogonalTransitionPair> getOrthogonalPairs() {
+        List<OrthogonalTransitionPair> orthogonalTransitionPairs = new ArrayList<>();
+        List<Transition> transitions = getAllTransitions();
+
+        for(int t1Index = 0; t1Index < transitions.size(); t1Index++) {
+            Transition t1 = transitions.get(t1Index);
+            for(int t2Index = t1Index+1; t2Index < transitions.size(); t2Index++) {
+                Transition t2 = transitions.get(t2Index);
+                State t1Src = this.statesMap.get(t1.fromStateName.replace("/", "_"));
+                State t1Dest = this.statesMap.get(t1.toStateName.replace("/", "_"));
+                State t2Src = this.statesMap.get(t2.fromStateName.replace("/", "_"));
+                State t2Dest = this.statesMap.get(t2.toStateName.replace("/", "_"));
```

Comment:
Why do we need to replace the "/" with "_" here? it should have already been replaced in the constructor of the transition.
![image](https://user-images.githubusercontent.com/40646603/223289298-ce0369b5-a12f-4fe6-aa27-1adb2455ba78.png)